From 3c1a1ccdc4d4dcc1e698041c8c4c1e3bd27d5cef Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@renesas.com>
Date: Tue, 8 Mar 2022 14:24:23 +0700
Subject: [PATCH] mmngr: Add MM_OMXBUF_SIZE for RZG2L, RZG3L, RZG3E and RZV2H
 OMX

This commit adds:
    MM_OMXBUF_SIZE (128 MB) for RZ/G2L Series and RZ/G3L
    MM_OMXBUF_SIZE (256 MB) for RZ/G3E and RZ/V2H

Signed-off-by: trungvanle <trung.le.xk@renesas.com>
---
 .../files/mmngr/include/mmngr_private.h        | 18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h b/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
index 56ab1f6..0409591 100644
--- a/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
+++ b/mmngr_drv/mmngr/mmngr-module/files/mmngr/include/mmngr_private.h
@@ -1,7 +1,7 @@
 /*************************************************************************/ /*
  MMNGR
 
- Copyright (C) 2015-2019 Renesas Electronics Corporation
+ Copyright (C) 2015-2025 Renesas Electronics Corporation
 
  License        Dual MIT/GPLv2
 
@@ -204,6 +204,22 @@ static void mm_exit(void);
 
 static int validate_memory_map(void);
 
+#ifdef MMNGR_RZG2L
+#define MM_OMXBUF_SIZE		(128 * 1024 * 1024)
+#endif /* MMNGR_RZG2L */
+
+#ifdef MMNGR_RZG3L
+#define MM_OMXBUF_SIZE		(128 * 1024 * 1024)
+#endif /* MMNGR_RZG3L */
+
+#ifdef MMNGR_RZG3E
+#define MM_OMXBUF_SIZE		(256 * 1024 * 1024)
+#endif /* MMNGR_RZG3E */
+
+#ifdef MMNGR_RZV2H
+#define MM_OMXBUF_SIZE		(256 * 1024 * 1024)
+#endif /* MMNGR_RZV2H */
+
 #ifdef MMNGR_SALVATORX
 #define MM_OMXBUF_SIZE		(256 * 1024 * 1024)
 #endif /* MMNGR_SALVATORX */
-- 
2.34.1

